.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+### Closes: 
+<*reference the related issue e.g. #150*>
+
+### Description:
+<*Add a description of the changes proposed in the pull request.*>
+
+### Preview:
+<*insert your gif/screenshot/loom recording here*>
+
+### Scope of the changes (select all that apply):
+- [ ] Frontend  
+- [ ] Backend  
+- [ ] Common  
+- [ ] E2E tests  
+- [ ] Docs  
+- [ ] CI/CD  
+
+### To do:
+- [ ] update Changelog  
+- [ ] ...
```

Comment:
I think we should remove this section for now. We no longer need to update the changelog in each PR.